tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,579 @@
+{
```

Comment:
<p>The first sentence is incomplete</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>